app/institutions/dashboard/-components/institutional-users-list/styles.scss

```diff
@@ -0,0 +1,94 @@
+.select {
+    max-width: 320px;
+    padding: 7px 16px 7px 14px;
+    margin-bottom: 15px;
+    border-color: #ddd;
+    border-radius: 2px;
+    color: #337ab7;
+}
+
+.table {
+    margin-bottom: 45px;
+
+    table {
+        width: 100%;
+        margin-bottom: 15px;
+        table-layout: fixed;
+        box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+    }
+
+    th,
+    td {
+        padding: 15px;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+    }
+
+    :global(.ember-content-placeholders-text__line) {
+        margin-bottom: 0;
+    }
+}
+
+.header {
+    th {
+        background: #365063;
+        border: 0;
+        color: #fff;
+        text-transform: uppercase;
+        vertical-align: middle;
+    }
+
+    .nested-header {
+        padding: 0 15px;
+        text-align: center;
+    }
+}
+
+.item {
+    td {
+        border-right: 1px solid #ddd;
+        border-top: 0;
+
+        &:nth-child(3),
+        &:nth-child(4) {
```

Comment:
This works fine for now, but there are plans to add additional columns to this table. I'd rather have a specific class on the items that need centering.

Reply:
Sorry, I mean it works, but we should change it for a bit of future-proofing. 😄